pkg/client/resmgmt/resmgmt.go

```diff
@@ -872,8 +874,8 @@ func checkRequiredCCProposalParams(channelID string, req InstantiateCCRequest) e
 		return errors.New("must provide channel ID")
 	}
 
-	if req.Name == "" || req.Version == "" || req.Path == "" || req.Policy == nil {
-		return errors.New("Chaincode name, version, path and policy are required")
+	if req.Name == "" || req.Lang == "" || req.Version == "" || req.Path == "" || req.Policy == nil {
+		return errors.New("Chaincode name, language, version, path and policy are required")
```

Comment:
Generally, we try not to break the existing client code. Language should be optional and default to golang.

Reply:
Hi, has been modified to golang as the default chaincode type.

Reply:
Should also revert the client code (tests) where "golang" was added, we should try not to change old client code if API is not broken. There should be at least one test with "golang" though.

Reply:
Hi, Unit tests have been added.

Reply:
Great, just need to revert the client code (tests where "golang" option was added) to the original, as this PR is not breaking it. I added comments below.